### ISSUE TITLE
The result is different from your paper

### ISSUE THREAD

**Issue by hurricane2018** [DagnyT/hardnet#13]
I use your code to test Notredame and Yosemite using Yosemite dataset. Here is the result.

Train Epoch: 9 [4997120/5000000 (100%)]	Loss: 0.645969: : 4883it [16:58,  4.80it/s]
notredame Test Epoch: 9 [92160/100000 (92%)]: : 98it [00:40,  2.43it/s]
Test set: Accuracy(FPR95): 0.00488000

yosemite Test Epoch: 9 [92160/100000 (92%)]: : 98it [00:42,  2.30it/s]
Test set: Accuracy(FPR95): 0.01864000

The test result is better than that in your paper. So when you wrote your paper, did you average several tests ?

**Comment by hurricane2018** [comment]
I just run the HardNet.py using the default parameter.

**Comment by ducha-aiki** [comment]
Have you read an updated arXiv version of the paper?
"Post-NIPS update. After two major updates of PyTorch library, we were not able to reproduce the results. Therefore, we did a hyperparameter search and we found that increasing the learning rate to 10 and dropout rate to 0.3 leads to better results –see Figure 5 and Table 1. We have obtained the same results on different machines"

Regarding Brown dataset results, they are quite noisy because of wrong labels and saturation. HpPatches
